ARTstor.js

```diff
@@ -1,22 +1,25 @@
 {
-	"translatorID": "5278b20c-7c2c-4599-a785-12198ea648bf",
-	"label": "ARTstor",
-	"creator": "Sebastian Karcher",
-	"target": "^https?://library\\.artstor\\.org",
-	"minVersion": "3.1",
-	"maxVersion": "",
```

Comment:
Yes, the above pattern works fine. 

I put in the pattern (https?://([^/]+\.)?(artstor|sscommons)\.org\/(open)?library) without double slash (\\.) in scaffold, and it works. 

Does that mean I should put in extra (\) in the code?